gallery/src/PageHeader/PageHeader.tsx

```diff
@@ -4,37 +4,52 @@
  * the terms of the Eclipse Public License 2.0 which accompanies this
  * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
  */
-import React from 'react';
+import React, { useState } from 'react';
+
+import { NxButton } from '@sonatype/react-shared-components';
 
 import HostedVersionsSelect from './HostedVersionsSelect';
+import DarkModeModal from '../DarkModeModal/DarkModeModal';
 
 const plaidVillain = require('./plaid-villain.svg');
 const tartanLogotype = require('./tartan-logotype.svg');
 
 import './PageHeader.scss';
 
-const PageHeader = () => (
-  <header className="nx-page-header gallery-page-header">
-    <div className="nx-page-header__inner">
-      <a className="gallery-page-header__home-link"
-         href="#/">
-        <img className="gallery-page-header__logo-image"
-             src={plaidVillain}
-             alt="" />
-        <img className="gallery-page-header__logotype-image"
-             src={tartanLogotype}
-             alt="Sonatype Tartan" />
-      </a>
-
-      <span className="gallery-page-header__name">
-        React Shared Component Library
-      </span>
-
-      <div className="gallery-page-header__version">
-        <HostedVersionsSelect />
+const PageHeader = () => {
+  const [showModal, setShowModal] = useState(false);
+  const modalCloseHandler = () => setShowModal(false);
```

Comment:
I wonder if we could use `useToggle` here too for `setShowModal`, then we can pass in the `toggleDarkModeModal` fn directly into DarkModeModal modalCloseHandler prop below

Reply:
> I wonder if we could use `useToggle` here too for `setShowModal`, then we can pass in the `toggleDarkModeModal` fn directly into DarkModeModal modalCloseHandler prop below

Makes sense and cleaned things up nicely. Updated here: [861f2c6](https://github.com/sonatype/sonatype-react-shared-components/pull/782/commits/861f2c675042b530e22789e628967d0fbaa446c8)